src/worker.penumbra.js

```
@@ -51,46 +64,70 @@ class PenumbraWorker {
         }Truncating to common subset (${writablePorts.length}).`,
       );
     }
-    resources.forEach((resource, i) => {
-      if (!('url' in resource)) {
-        throw new Error(
-          'PenumbraDecryptionWorker.get(): RemoteResource missing URL',
-        );
-      }
-      const remoteStream = fromWritablePort(writablePorts[i]);
-      fetchAndDecrypt(resource).then((localStream) => {
-        localStream.pipeTo(remoteStream);
-      });
-    });
```

Bugfix: This swallows errors.